### PR TITLE
Implement new frontpage layout

### PR DESCRIPTION
**Pull request by pixeldesu**
The old frontpage was overcrowded with too much information, and everything combined in a relatively small layout. This PR revises the entire layout.

https://user-images.githubusercontent.com/1774242/162597136-6346262c-168b-4068-babf-71cd85745490.mp4

**Summarized changes:**
* New frontpage layout
  * No full-height header anymore
  * Alternating sections with more concise feature descriptions
  * Theming demonstration with a few hand-created themes
* Fixed a bug with theme application on the guest navbar
* Added locales for the new frontpage and removed the old locales

**Testing:**
* Take a look at the frontpage and give feedback

### CODE REVIEW

File: app/javascript/styles/components/_themes.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+}
+
+.theme-warning {
+  --background: #fcfaf4;
+  --primary: var(--warning);
+  --primary-text: 0, 0, 0;
+}
+
+.theme-danger {
+  --background: #fceff1;
```
[the remark this answers is not in the document]
Comment (pixeldesu) [reply app/javascript/styles/components/_themes.scss:13]: This file is the only place this color is used.

File: app/javascript/styles/components/_themes.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
+.theme-success {
+  --background: #f4fcf6;
+  --primary: var(--success);
+}
+
+.theme-warning {
+  --background: #fcfaf4;
```
[the remark this answers is not in the document]
Comment (pixeldesu) [reply app/javascript/styles/components/_themes.scss:7]: This file is the only place this color is used.

File: app/javascript/styles/components/_themes.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.theme-success {
+  --background: #f4fcf6;
```
[the remark this answers is not in the document]
Comment (pixeldesu) [reply app/javascript/styles/components/_themes.scss:2]: This file is the only place this color is used.

**Review by nilsding — APPROVED** [review]
Fancy shit!

Two thoughts came to my mind... do with them what you think is right 🙂 

- the text reads like it's straight from an informercial
- instead of having those big ass icons maybe we can have our raccoon there (e.g. for the theming one have them paint over a simplified retrospring profile with a paintbrush)

**Review by raccube — APPROVED** [review]
Looks alright. If we don't get raccoon graphics we could scatter some other icons around near the big ones.

**Pull request merged**